<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarkupWindow</class>
 <widget class="QMainWindow" name="MarkupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1279</width>
    <height>774</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MarkupTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="13">
     <widget class="MarkupView" name="markupView">
      <property name="cursor" stdset="0">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::NoDrag</enum>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::AnchorUnderMouse</enum>
      </property>
      <property name="resizeAnchor">
       <enum>QGraphicsView::NoAnchor</enum>
      </property>
      <property name="viewportUpdateMode">
       <enum>QGraphicsView::FullViewportUpdate</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="brightLabel">
      <property name="text">
       <string>Bright</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="pointsSize">
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>0.500000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="curve">
      <property name="minimumContentsLength">
       <number>3</number>
      </property>
      <item>
       <property name="text">
        <string>Left eye</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Right eye</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>mouth</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="ready">
      <property name="text">
       <string>Ready: </string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="searchFile">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSlider" name="scaleOnSegment">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QCheckBox" name="scaleSave">
      <property name="text">
       <string>Save scale</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSlider" name="contrast">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="singleStep">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QCheckBox" name="allowEdit">
      <property name="text">
       <string>Allow edit</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QListWidget" name="searchedFiles">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <property name="currentRow">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="contrastLabel">
      <property name="text">
       <string>Contrast</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="scaleLabel">
      <property name="text">
       <string>Scale on segment</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSlider" name="bright">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1279</width>
     <height>22</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="loadFacesAction"/>
    <addaction name="actionLoad_shapes"/>
    <addaction name="actionLoad_shapes_as"/>
    <addaction name="actionUpdate_from_JSON"/>
    <addaction name="actionLoad_shape"/>
    <addaction name="actionSave_shape"/>
    <addaction name="actionSave_shapeas"/>
    <addaction name="separator"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuControl">
    <property name="title">
     <string>Control</string>
    </property>
    <addaction name="actionNext"/>
    <addaction name="actionPast"/>
   </widget>
   <widget class="QMenu" name="menuLadnmarks">
    <property name="title">
     <string>Landmarks</string>
    </property>
    <addaction name="actionDelete"/>
    <addaction name="actioninvisible"/>
    <addaction name="actionNext_invisible_segment"/>
    <addaction name="actionPast_invisible_segment"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuControl"/>
   <addaction name="menuLadnmarks"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="loadFacesAction">
   <property name="text">
    <string>Load faces</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionPast">
   <property name="text">
    <string>Past</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionLoad_shape">
   <property name="text">
    <string>Load shape as..</string>
   </property>
  </action>
  <action name="actionSave_shape">
   <property name="text">
    <string>Save shape</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionUpdate_from_JSON">
   <property name="text">
    <string>Load shape</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSave_shapeas">
   <property name="text">
    <string>Save shape as..</string>
   </property>
  </action>
  <action name="actionLoad_shapes_as">
   <property name="text">
    <string>Load shapes as..</string>
   </property>
  </action>
  <action name="actionLoad_shapes">
   <property name="text">
    <string>Load shapes</string>
   </property>
  </action>
  <action name="actioninvisible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Invisible segment</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionNext_invisible_segment">
   <property name="text">
    <string>Next invisible segment</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionPast_invisible_segment">
   <property name="text">
    <string>Past invisible segment</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MarkupView</class>
   <extends>QGraphicsView</extends>
   <header>markupview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
